cmake/version.h.in

```diff
@@ -38,6 +38,6 @@
 #cmakedefine KOVRI_CODENAME "@KOVRI_CODENAME@"
 
 #cmakedefine I2P_VERSION "@I2P_VERSION@"
-#cmakedefine I2P_NETWORK_ID "@I2P_NETWORK_ID@"
+#cmakedefine I2P_NETWORK_ID @I2P_NETWORK_ID@
```

Comment:
I see that you want this for the below cli option but if, after reading my below comment, you *don't* decide to keep the option, then I think this should be reverted since the only implementation purpose is string.